src/test/scala/treadle/VecSpec.scala

```diff
@@ -31,20 +33,13 @@ class VecSpec extends FreeSpec with Matchers {
         |
       """.stripMargin
 
-    val optionsManager = new TreadleOptionsManager {
-      treadleOptions = treadleOptions.copy(
-        writeVCD = false,
-        setVerbose = false,
-        showFirrtlAtLoad = false,
-        rollbackBuffers = 20
-      )
-      commonOptions = commonOptions.copy(
-        targetDirName = "test_run_dir/vec_spec1",
-        topName = "vec_spec"
-      )
-    }
+    val options = Seq(
+      WriteVcdAnnotation,
+      RollBackBuffersAnnotation(20),
+      TopNameAnnotation("vec_spec_1")
```

Comment:
Do you want a `TargetDirAnnotation` here?

Reply:
No, I have fixed the internal treadle tests harness to default to test_run_dir, keeps the root cleaner